libs/core/tests/unit_tests/utils/test_function_calling.py

```diff
@@ -295,6 +295,27 @@ def test_convert_to_openai_function(
     runnable_expected["parameters"] = parameters
     assert actual == runnable_expected
 
+    # Test simple Tool
+    def my_function(input_string: str) -> str:
```

Comment:
Do we have a check that if we did this with structured tool / decorator it actually uses the argument name?

For prompting purposes it matters

Reply:
Might not be following but is this what you're after? https://github.com/langchain-ai/langchain/pull/26327/commits/84322fda9cd0f0e98041165ae8ddcc6d7d3a4ed0